migrations/0007-create-exhibitions.js

```diff
@@ -0,0 +1,240 @@
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['exhibitionChapterPage']
+        }
+      ],
+
+      linkType: 'Entry'
+    });
+
+  exhibitionPage
+    .createField('credits')
+    .name('Credits')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+  exhibitionPage.changeFieldControl('name', 'builtin', 'singleLine', {});
+
+  exhibitionPage.changeFieldControl('identifier', 'builtin', 'slugEditor', {
+    helpText: 'Will always be prefixed with "/exhibition/"'
+  });
+
+  exhibitionPage.changeFieldControl('headline', 'builtin', 'singleLine', {
+    helpText:
+      'For the secondary text over the hero banner only, appears under the title.'
+  });
+
+  exhibitionPage.changeFieldControl('description', 'builtin', 'singleLine', {});
+  exhibitionPage.changeFieldControl('text', 'builtin', 'markdown', {});
+  exhibitionPage.changeFieldControl(
+    'primaryImageOfPage',
+    'builtin',
+    'entryLinkEditor',
+    {}
+  );
+
+  exhibitionPage.changeFieldControl('hasPart', 'builtin', 'entryCardsEditor', {
+    bulkEditing: false
+  });
+
+  exhibitionPage.changeFieldControl('credits', 'builtin', 'markdown', {});
+  const exhibitionChapterPage = migration
+    .createContentType('exhibitionChapterPage')
+    .name('Exhibition Chapter Page')
+    .description(
+      'An Exhibition Chapter, consisting of multiple content sections. Relates to sibling chapters through a parent Exhibition Page.'
+    )
+    .displayField('name');
+
+  exhibitionChapterPage
+    .createField('name')
+    .name('Title')
+    .type('Symbol')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  exhibitionChapterPage
+    .createField('identifier')
+    .name('URL slug')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        prohibitRegexp: {
+          pattern: 'credits',
+          flags: null
+        },
+
+        message:
+          'You may not use the slug \'credits\' as this page will be automatically generated. Set contents for the credits page on the exhibiton itself.'
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  exhibitionChapterPage
+    .createField('primaryImageOfPage')
+    .name('Hero Banner')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkContentType: ['heroBanner']
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  exhibitionChapterPage
+    .createField('hasPart')
+    .name('Sections')
+    .type('Array')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['cardGroup', 'embed', 'richText']
```

Comment:
Should this also include `imageWithAttribution` for the inclusion of attributed images not used as the hero?